### PR TITLE
Docs: Added contributing.md

### PR DESCRIPTION
**Pull request by Abhinavcode13**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Thank you for your interest in contributing to the Mojafos repository! Your contributions are important and will help to improve the project for everyone. Before you begin, please consider the guidelines below.
+
+## Getting Started
+
+- Make sure you have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed on your machine.
+- Fork the repository and clone it locally.
+  
+  ```
+  git clone https://github.com/<your_github_username>/mojafos.git
```
Comment (elijah0kello) [inline CONTRIBUTING.md:11]: Please change this username to openMF.

File: README.md [hunk 2]
```diff
@@ -193,7 +193,8 @@ This will delete all resources in the created namespaces and if the kubernetes c
 Please note that cleaning up the resources will take some time.
 
 ## CONTRIBUTION
- TBD
+
+Find the contributing guidelines here[https://github.com/openMF/mojafos/blob/master/CONTRIBUTING.md]
```
Comment (elijah0kello) [inline README.md:197]: Kindly fix this link. It does not render well

**Review by elijah0kello — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Abhinavcode13) [reply CONTRIBUTING.md]: Resolved

File: README.md (hunk 2 above)
Comment (Abhinavcode13) [reply README.md:197]: Once this PR is merged, everything will function as intended.

Comment (Abhinavcode13) [reply README.md:197]: Similar to how README.md contains the route https://github.com/openMF/mojafos/blob/master/README.md, CONTRIBUTING.md will have a similar link with the file name changed at the end.

File: README.md [hunk 3]
```diff
@@ -193,7 +193,8 @@ This will delete all resources in the created namespaces and if the kubernetes c
 Please note that cleaning up the resources will take some time.
 
 ## CONTRIBUTION
- TBD
+
+Find the contributing guidelines [here](./CONTRIBUTING.md)
```
Comment (elijah0kello) [inline README.md:197]: I just changed this part to a relative path and fixed the fromating

**Review by elijah0kello — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Abhinavcode13) [reply README.md:197]: LGTM

**Pull request merged**